kbase-extension/static/narrative_paths.js

```diff
@@ -122,7 +122,7 @@ require.config({
         'kbaseNarrativeDataPanel': 'kbase/js/widgets/narrative_core/kbaseNarrativeDataPanel',
         'kbaseNarrativeDataList': 'kbase/js/widgets/narrative_core/kbaseNarrativeDataList',
         'kbaseNarrativeSidePanel': 'kbase/js/widgets/narrative_core/kbaseNarrativeSidePanel',
-        'kbaseNarrativeJobsPanel': 'kbase/js/widgets/narrative_core/kbaseNarrativeJobsPanel',
+        'jobCommChannel': 'kbase/js/widgets/narrative_core/jobCommChannel',
```

Comment:
kbaseNarrativeJobsPanel is replaced with a jobCommChannel! Release notes worthy!

Reply:
Agreed!